build.sbt

```diff
@@ -61,6 +61,6 @@ lazy val azure = project
       "ch.timo-schmid" %% "slf4s-api" % "1.7.26",
       "io.netty" % "netty-all" % NettyVersion,
       "com.microsoft.azure" % "azure-storage-blob" % "10.5.0",
-      "com.azure" % "azure-identity" % "1.0.0",
+      "com.azure" % "azure-identity" % "1.1.0-beta.3",
```

Comment:
Did you test this with a destination in SDBE?
It's a bit of a hassle to test this but due to the dependency update there may be runtime incompatibilities that are not detected here. 

Happy to help with setting up the dev env for this, if needed.

Reply:
yes i did